### DISCUSSION TITLE
Playlist additional features

### DISCUSSION THREAD

**Discussion by SensouShin** [ppy/osu#17491] [Ideas]
Regarding playlist creation: 
set room visibility by country so country specific playlists can be made (it can increase the competitiveness in smaller countries)

Regarding playlist Screen: This screen is already saturated with things but i think there are some missing feature like

1. current score on maps (yes you can view the score with the "view result" button, but you dont have any overall idea which map have you played already or what score do you have on it. you have to check them one by one)
2. Minimal information about beatmaps like: AR, BPM, SR (I'm aware you can view the SR by hovering over the colored circle but not the greatest solution imo.)

**Comment by bdach** [comment]
point 2 previously suggested in https://github.com/ppy/osu/discussions/16008

**Reply by SensouShin** [reply]
I'm not the greatest fan of the hover over solution but i can live with it. The other two point still remains.

**Reply by Theighlin** [reply]
Point 1 suggested in #9762

**Reply by SensouShin** [reply]
> Point 1 suggested in #9762
I think we are not on the same page.I'm talking about this screen:
![image](https://user-images.githubusercontent.com/6898013/160391748-04c5ea34-2cea-47e5-a9aa-1583f4334bef.png)
when you look at the playlist you can't tell  which map you played or what score do you have on them

**Reply by Theighlin** [reply]
I pointed to that because i thought it was very close  in what you meant (by clicking on your total score you could see the maps that you played and their score, not one by one).

What you say then will probably happen with the playlist screen redesign.

**Reply by SensouShin** [reply]
> I pointed to that because i thought it was very close in what you meant (by clicking on your total score you could see the maps that you played and their score, not one by one).
> 
> What you say then will probably happen with the playlist screen redesign.

Yeah you right. I'm thinking something like this:
![plscore](https://user-images.githubusercontent.com/6898013/160396458-0d0efc5c-9c42-4b3a-a1fa-ec486f703f51.png)

Also probably some indication of FC, something else than ranking, because the S ranking on a map doesn't mean anything in this version of lazer scoring
